frontend/EmpUI/src/services/task-service.ts

```
@@ -43,7 +43,7 @@ export const taskService = baseService.injectEndpoints({
     }),
     deleteTask: builder.mutation<void, number>({
       query: (taskId) => ({
-        url: `${API_END_POINTS.tasks}?Id=${taskId}`,
+        url: API_END_POINTS.tasks + `/${taskId}`,
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)